config/initializers/session_store.rb

```diff
@@ -1,4 +1,4 @@
-if ENV['RAILS_USE_MEMORY_STORE'] || (!Rails.env.development? && !Rails.env.production?)
+if ENV['RAILS_USE_MEMORY_STORE'] || TokenStore.use_memory?
```

Comment:
I mean, I see that literally the code is the same, but this change couples TokenStore (which is for API tokens and I guess now WS tokens, to UI Sessions, which they have nothing to do with one another.  If someone changes the TokenStore.use_memory? method, they would never expect that it would affect UI Session setup.